tests/test_ht_spkitable.c

```diff
@@ -1,391 +1,450 @@
 /*
  * This file is part of RTRlib.
  *
- * RTRlib is free software; you can redistribute it and/or modify
- * it under the terms of the GNU Lesser General Public License as published by
- * the Free Software Foundation; either version 3 of the License, or (at your
- * option) any later version.
+ * This file is subject to the terms and conditions of the MIT license.
+ * See the file LICENSE in the top level directory for more details.
  *
- * RTRlib is distributed in the hope that it will be useful, but
- * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
- * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public
- * License for more details.
- *
- * You should have received a copy of the GNU Lesser General Public License
- * along with RTRlib; see the file COPYING.LESSER.
+ * Website: http://rtrlib.realmv6.org/
  */
+
 #include "rtrlib/spki/hashtable/ht-spkitable.h"
 #include <stdio.h>
 #include <string.h>
 #include <time.h>
 #include <stdlib.h>
 
-static struct spki_record *create_record(int ASN, int ski_offset, int spki_offset, struct rtr_socket *socket);
-
-static bool compare_spki_records(struct spki_record *r1, struct spki_record *r2)
+/* test_ht_4 */
+#define NUM_TABLE_X		(50)
+#define NUM_TABLE_Y		(50)
+/* test_ht_6 */
+#define NUM_SKIS		(10)
+#define NUM_SKIS_RECORDS	(10)
+/* test_ht_7 */
+#define NUM_OF_SKI		(200)
+#define NUM_OF_RECORDS		(2000)
+
+static struct spki_record *create_record(int ASN,
+					 int ski_offset,
+					 int spki_offset,
+					 struct rtr_socket *socket);
+/**
+ * @brief Compare SPKI records for equality
+ *
+ * @return true if r1 == r2, false otherwise
+  */
+static bool compare_spki_records(struct spki_record *r1,
+				 struct spki_record *r2)
 {
-    if(r1->asn != r2->asn) {
-        return false;
-    }
-    if(r1->socket != r2->socket) {
-        return false;
-    }
-    if(memcmp(r1->ski, r2->ski, SKI_SIZE) != 0) {
-        return false;
-    }
-    if(memcmp(r1->spki, r2->spki, SPKI_SIZE) != 0) {
-        return false;
-    }
-    return true;
+	if (r1->asn != r2->asn)
+		return false;
+	if (r1->socket != r2->socket)
+		return false;
+	if (memcmp(r1->ski, r2->ski, SKI_SIZE) != 0)
+		return false;
+	if (memcmp(r1->spki, r2->spki, SPKI_SIZE) != 0)
+		return false;
+
+	return true;
 }
 
-static struct spki_record *create_record(int ASN, int ski_offset, int spki_offset, struct rtr_socket *socket)
+/**
+ * @brief Create a SPKI record
```

Comment:
The description doesn't provide any additional value compared to the informations you get out of the function signature.